### PR TITLE
<Ellipsis/> - [internal] - New component to replace EllipsisHOC

### PR DESCRIPTION
**Pull request by EgoziE**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5622/

File: src/common/Ellipsis/Ellipsis.st.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  overflow: hidden;
+  height: max-content;
+}
+
+.tooltip::popoverElement {
+  max-width: 100%;
+}
+
+/* Tooltip content */
+.content {
```
Comment (shlomitc) [inline src/common/Ellipsis/Ellipsis.st.css:34]: remove

File: src/common/Ellipsis/Ellipsis.st.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+:import {
+  -st-from: "../../Tooltip/Tooltip.st.css";
+  -st-default: Tooltip;
+}
+
+:import {
```
Comment (shlomitc) [inline src/common/Ellipsis/Ellipsis.st.css:6]: remove

**Comment by EgoziE** [comment]
Current failing visual tests should be accepted.

**Pull request merged**